### PR TITLE
Backpacking

### PR DESCRIPTION
**Pull request by Blundir**
## About The Pull Request

Adds two new MOLLE backpacks, sprites made by Crow#7854 .

Reuses a lot of unused backpacks for general population, green normal satchel is available to anybody, medical ones to corpsmen, industrial ones to engis.

dreammaker_VKTKbh6TIQ

## Why It's Good For The Game

More drip is always good.

Here is a demonstration of the two new packs.
image

## Changelog

:cl:
add: added two new MOLLE backpacks
add: reused a lot of almost unused backpacks and satchels
fix: fixed corpsman backpack using old cm sprites in some situations
imageadd: new MOLLE packs sprites
imageadd: added some backpack inhands sprites
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
image

**Comment by QualityVan** [comment]
These have the same capabilities as the current options in terms of storage, yeah?

**Comment by Blundir** [comment]
> These have the same capabilities as the current options in terms of storage, yeah?

These are reskins, nothing else

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -266,6 +270,12 @@
 	name = "\improper lightweight IMP backpack"
 	desc = "The standard-issue pack of the TGMC forces. Designed to slug gear into the battlefield."
 
+/obj/item/storage/backpack/marine/standard/molle
+	name = "\improper T16 MOLLE Backpack"
```
Comment (XSlayer300) [inline code/game/objects/items/storage/backpack.dm:274]:
```suggestion
	name = "\improper T16 MOLLE backpack"
```
stop it. get some help

File: code/game/objects/items/storage/backpack.dm [hunk 2]
```diff
@@ -373,8 +383,14 @@
 	max_storage_space = 15
 	access_delay = 0
 
+/obj/item/storage/backpack/marine/satchel/molle
+	name = "\improper T13 MOLLE Satchel"
```
Comment (XSlayer300) [inline code/game/objects/items/storage/backpack.dm:387]:
```suggestion
	name = "\improper T13 MOLLE satchel"
```

File: code/game/objects/items/storage/backpack.dm [hunk 3]
```diff
@@ -373,8 +383,14 @@
...
+	desc = "The latest satchel developed by Crowford Armory Union on the military order of TGMC. Thanks to the introduction of new MOLLE fastening systems, it turned out to beltbags and backpacks that are not inferior to roominess and portable weight, while also reducing the size of backpacks that have gone to hang from the back on the belt."
+	icon_state = "MOLLEbeltbag"
+	item_state = "MOLLEbeltbag"
+
 /obj/item/storage/backpack/marine/satchel/green
-	name = "\improper TGMC satchel"
+	name = "\improper Green TGMC satchel"
```
Comment (XSlayer300) [inline code/game/objects/items/storage/backpack.dm:393]:
```suggestion
	name = "\improper green TGMC satchel"
```

**Review by Kuro020 — APPROVED** [review]

**Comment by Lumipharon** [comment]
I think it's better to leave some items that clearly look civilian (like those med packs), to actual civilians. Marines can loot then groundswell, it just looks out of place as standard kit.

**Comment by Blundir** [comment]
dmi merge conflicts, my beloved

**Pull request closed without merging**